ca_bc/people.py

```diff
@@ -52,12 +52,15 @@ def scrape(self):
         members = data["data"]["allMemberParliaments"]["nodes"]
 
         assert len(members), "No members found"
+        print(len(members))
         for member in members:
             image = "https://lims.leg.bc.ca/public" + member["image"]["path"]
             district = member["constituency"]["name"]
             name = member["member"]["firstName"] + " " + member["member"]["lastName"]
             party = member["party"]["name"]
 
+            name = name.replace(":", "").replace("'","")
```

Comment:
This is not an appropriate fix. Those characters are part of the person's name in this exceptional case.

The correct fix is 99beae01.